platform-includes/session-replay/setup/javascript.angular.mdx

```diff
@@ -22,20 +22,11 @@ Sentry.init({
 });
 ```
 
-With the settings above, session replays with errors are always captured.
-You can verify this by triggering a test error somewhere in your Angular app, for example in your main app component:
+### Verify
 
-```html {filename: app.component.html}
-<button (click)="throwTestError()">Test Sentry Error</button>
-```
-
-Then, in your `app.component.ts` add:
+While you're testing, we recommend that you set `replaysSessionSampleRate` to `1.0`. This ensures that every user session will be sent to Sentry.
 
-```javascript {filename: app.component.ts}
-public throwTestError(): void {
-  throw new Error("Sentry Test Error");
-}
-```
+Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping `replaysErrorSampleRate` set to `1.0`.
```

Comment:
```suggestion
Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping `replaysOnErrorSampleRate` set to `1.0`.
```

Reply:
```suggestion
Once testing is complete, **we recommend lowering this value in production**. We still recommend keeping `replaysOnErrorSampleRate` set to `1.0`, so that, whenever possible, every error has an associated replay with additional debugging context.
```
I'd add something like this (feel free to adjust) to explain why it should still be set to 1 and to clarify that this is a different option than above (at first read, I thought `replaysSessionSampleRate` and  `replaysOnErrorSampleRate` were the same thing).